DigitalLearningSolutions.Web/Styles/myAccount/myAccount.scss

```diff
@@ -15,3 +15,25 @@
   justify-content: space-between;
   flex-wrap: wrap;
 }
+
+.account-switch-warning {
+  margin-bottom: 15px;
+}
+
+.account-switch-warning-msg {
+  background-color: #f7e178;
+  margin-left: 20px;
+  margin-bottom: 5px;
+  padding: 5px;
+  width: 55%;
+  text-align: center;
+}
+
+.account-switch-warning-msg-description {
+  background-color: #f9efb0;

```

Comment:
As above.

Reply:
Rather than implementing this with new styles. Can we use the [NHS Design System warning callout component](https://service-manual.nhs.uk/design-system/components/warning-callout) that already exists in our scss?